### DISCUSSION TITLE
Vulnerability Detected CVE-2021-43529 for nss 3.49.0-4

### DISCUSSION THREAD

**Discussion by martinzeifang** [wazuh/wazuh#13570] [Q&A] [answered]
Hi Everyone, 
the Vulnerability Scanner detects a vulnarability CVE-2021-43529 for 3.49.0-4. I'm running wazuh from the ova with centos 7.9 that only supports ns 3.67.0-4.el7_9. 
![image](https://user-images.githubusercontent.com/59862886/169966463-6174e1ae-5692-4a59-a6be-c64bdd6b5e37.png)

does wazuh use that vulnerable part of nss? can I exclude that finding or does anyone know a solution to patch nss for centos 7.9?

thanks everyone

**Comment by carlosdams** [comment, marked as answer]
Hi @martinzeifang ,
Thanks for using Wazuh!

- Does wazuh use that vulnerable part of nss? 

nss comes with CentOS 7.9.

- Can I exclude that finding or does anyone know a solution to patch nss for centos 7.9?

It is not necessary to update or patch nss based on your current version nss 3.67.0-4.el7_9.
If you checked the link listed in the screenshot: https://bugzilla.redhat.com/show_bug.cgi?id=2024370 it will take you to CVE-2021-43527.
This is fixed for your current version of nss, verifiy it by executing the following: `rpm -q --changelog nss | grep -B 1 CVE-2021-43527`

[CVE-2021-43529](https://bugzilla.redhat.com/show_bug.cgi?id=2088353) is linked to CVE-2021-43527 because the latter is the original security issue, however, it affects more precisely Thunderbird. More information from redhat here: https://access.redhat.com/security/cve/cve-2021-43529 
You can check if you have vulnerable software related to this vulnerability using this [script from redhat](https://access.redhat.com/sites/default/files/cve-2021-43527--2021-12-01-1901.sh) and will also let you know if you have Thunderbird.
Therefore, you can safely ignore that finding based on your nss version 3.67.0-4.el7_9.

I hope this information address your concerns, let me know please.
